From fcd4906958c69b2c3cb835600da1288731788c60 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 26 Nov 2018 08:31:08 +0900
Subject: [PATCH 457/909] arm64: dts: renesas: ulcb: sort node label

commit 6d7b56e3403bd0fa7a59fa1102f192d4e5aab175 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch sorts the node label to improve maintainability.
The sort has been done alphabetically with the node label name as the key.

This patch does not include functional changes.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/ulcb.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb.dtsi b/arch/arm64/boot/dts/renesas/ulcb.dtsi
index 89daca7356df..0f96ee21247d 100644
--- a/arch/arm64/boot/dts/renesas/ulcb.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb.dtsi
@@ -398,6 +398,11 @@
 	};
 };
 
+&rwdt {
+	timeout-sec = <60>;
+	status = "okay";
+};
+
 &scif2 {
 	pinctrl-0 = <&scif2_pins>;
 	pinctrl-names = "default";
@@ -447,8 +452,3 @@
 
 	status = "okay";
 };
-
-&rwdt {
-	timeout-sec = <60>;
-	status = "okay";
-};
-- 
2.17.1

